src/charts/chartOptions.js

```diff
@@ -6,22 +6,22 @@ define(['jquery', 'common/rivetsExtra', "charts/chartWindow", "charts/charts", '
 
     var state = [], view = [], template_manager = {}, stringWidth= {}, isListenerAdded = false;
 
-    var timeperiod_arr = [{value: "1t", name: "1 Tick", digit: 1, type: "ticks"}, 
```

Comment:
@negar-binary This file now contains some white spaces which is confusing because in diff its bit hard to find out what lines were changed for review